actions/actions.py

```diff
@@ -814,3 +814,24 @@ def run(self, dispatcher, tracker, domain) -> List[EventType]:
         tag_convo(tracker, label)
 
         return []
+
+
+class ActionWhyRasa(Action):
```

Comment:
`ActionWhyRasa` isn't being used anywhere in the stories. Can this custom action be removed?

Reply:
yes